configs/system/base.js

```diff
@@ -38,6 +38,19 @@ module.exports = {
         image: 'dashpay/sentinel:1.5.0',
       },
     },
+    tor: {
+      enable: true,
```

Comment:
I would keep it disabled for default and enable it in specific configs (testnet atm)